src/extensions/mvs/components/multilayer-color-theme.ts

```diff
@@ -137,11 +138,11 @@ export const MultilayerColorThemeName = 'mvs-multilayer';
 export function makeMultilayerColorThemeProvider(colorThemeRegistry: ColorTheme.Registry): ColorTheme.Provider<MultilayerColorThemeParams, typeof MultilayerColorThemeName> {
     return {
         name: MultilayerColorThemeName,
-        label: 'Multi-layer',
+        label: 'MVS Multilayer',
         category: ColorTheme.Category.Misc,
         factory: (ctx, props) => makeMultilayerColorTheme(ctx, props, colorThemeRegistry),
         getParams: (ctx: ThemeDataContext) => makeMultilayerColorThemeParams(colorThemeRegistry, ctx),
         defaultValues: DefaultMultilayerColorThemeProps,
-        isApplicable: (ctx: ThemeDataContext) => true,
+        isApplicable: (ctx: ThemeDataContext) => !!ctx.structure?.models.some(m => m.customProperties.has(MVSAnnotationsProvider.descriptor)),
```

Comment:
I would revert this change, makeMultilayerColorThemeProvider should be applicable even without MVSAnnotationsProvider.

(That's also the reason I didn't use the 'MVS ' prefix in the label, but I don't have a strong opinion about it)